src/openms_gui/include/OpenMS/VISUAL/SpectraTreeTab.h

```diff
@@ -92,13 +95,15 @@ namespace OpenMS
     QLineEdit* spectra_search_box_ = nullptr;
     QComboBox* spectra_combo_box_ = nullptr;
     TreeView* spectra_treewidget_ = nullptr;
+    LayerDataBase* layer_ = nullptr;
```

Comment:
why is this needed?
It seems to be assigned only once in the cpp and never read from.